Doc/library/os.rst

```diff
@@ -1171,6 +1171,10 @@ as internal buffering of data.
    current position; :const:`SEEK_END` or ``2`` to set it relative to the end of
    the file. Return the new cursor position in bytes, starting from the beginning.
 
+.. versionadded:: 3.3
+
+   Add support for :const:`SEEK_HOLE` and :const:`SEEK_DATA`.
```

Comment:
I think you need a `versionchanged` and not a `versionadded` (otherwise it means `lseek` was added in 3.3).
Also, you need to indent this block so that it apples to `lseek`.
```suggestion
   .. versionchanged:: 3.3

      Support added for :const:`SEEK_HOLE` and :const:`SEEK_DATA`.
```

Reply:
The original was `versionadded`, but `versionchanged` sounds more correct. I'll fix the indentation; thanks for noticing.

Reply:
Looking at the docs for `sqlite3.connect`, I see that `versionadded` is used for example when keyword parameters have been added. I suspect `versionadded` may be correct. cc. @CAM-Gerlach

Reply:
@AA-Turner: is `versionadded` correct?

Reply:
I think here it should be ``versionchanged``, as the acceptable values for ``whence`` were changed -- I'd tend to use [``versionadded``](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded) for a new parameter.

Reply:
Even for new parameters, it has been customary to use `versionchanged`. Witness for example the `subprocess.run` docs: https://docs.python.org/3/library/subprocess.html#subprocess.run

Reply:
I'll change it to `versionchanged`, then.